src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1149,6 +1149,31 @@ public static void setOverrideLoadConfig(Supplier<Config> loadConfig)
         exception
     }
 
+    /*
+     * How to pick a consensus protocol for CAS
+     * and serial read operations. Transaction statements
+     * will always run on Accord. Legacy in this context includes PaxosV2.
+     */
+    public enum LegacyPaxosStrategy
+    {
+        /*
+         * Default setting
```

Comment:
nit: I might not mention that this is the default right here. Could be a bit annoying to remember to change the doc later if the actual default changes.

Reply:
Removed mentioning the default.